install/terraform/modules/eks/eks.tf

```diff
@@ -47,6 +47,15 @@ resource "aws_security_group" "worker_group_mgmt_one" {
       "0.0.0.0/0",
     ]
   }
+  ingress {
```

Comment:
Same here, let's do terraform updates in it's own PR.

Reply:
Will remove